.github/workflows/test.yml

```diff
@@ -73,6 +73,12 @@ jobs:
           package_name: plotjuggler-fastdds-plugins
         if: always()
 
+      - name: Upload documentation
```

Comment:
Check this PR: https://github.com/eProsima/plotjuggler-fastdds-plugins/pull/25

Reply:
awaiting for confirmation. IMHO it is okay, as it is just a lightweight step. Probably it would be an overkill to create a new job to re-build everything (multiple heavy-weight steps) to just avoid the upload

Reply:
The idea of the job was to construct and test the documentation all-alone without building the rest of the project. But it cannot be done while the docs is a cmake subproject.
However, this computation will be done by github machines, and it will not be translated in a longer CI because windows will still be slower than ubunti. Thus, I think it is not a bad idea to do so.